src/odbc.cpp

```diff
@@ -359,25 +361,37 @@ class ConnectAsyncWorker : public ODBCAsyncWorker {
         sizeof(SQLSMALLINT),     // BufferLength
         NULL                     // StringLengthPtr
       );
+      // Some poorly-behaved drivers do not implement SQL_MAX_COLUMN_NAME_LEN,
```

Comment:
Perhaps "incomplete" or "non-compliant" driver would be better than "poorly behaved"

Reply:
Normally I'd agree, but the docs consistently use the term "behavior" to describe driver functionality. Also, it makes me smile to image these drivers as misbehaving little gremlins.

Will probably make a PR for https://github.com/mdbtools/mdbtools/ to bring SQLGetInfo into (better) compliance.